docs/getting-data/cloud-services.mdx

```diff
@@ -10,30 +10,39 @@ import SeattlePlaces from '!!raw-loader!@site/src/queries/athena/seattle_places.
 
 You can access and query Overture Maps parquet files directly in the cloud using one of the SQL query options documented below. You'll need an AWS account or a Microsoft Azure account.
 
+:::tip
+   Overture's S3 bucket is located in the us-west-2 AWS region, so use `us-west-2` for best performance.
+:::
+
 ## Amazon Athena (AWS)
 
-1. Add Overture as a cross-account data source using [these instructions](https://docs.aws.amazon.com/athena/latest/ug/data-sources-glue-cross-account.html).
-The Catalog ID is `913550007193`:
+### 1. Add Overture as a data source
 
-   - TIP: Overture's S3 bucket is located in the us-west-2 AWS region, so use Athena in `us-west-2` for best performance.
+Add Overture as a cross-account data source using [these instructions](https://docs.aws.amazon.com/athena/latest/ug/data-sources-glue-cross-account.html). The Catalog ID is `913550007193`.
```

Comment:
```suggestion
Add Overture as a cross-account data source using [these instructions](https://docs.aws.amazon.com/athena/latest/ug/data-sources-glue-cross-account.html). The Catalog ID is `913550007193` and the permissions policy required for querying is listed below.

<details>
  <summary>IAM Policy for Overture Glue catalog access</summary>
  <div>
```json
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Action": [
        "glue:GetTable*",
        "glue:GetDatabase*",
        "glue:GetPartition*"
      ],
      "Resource": [
        "arn:aws:glue:us-west-2:913550007193:catalog",
        "arn:aws:glue:us-west-2:913550007193:database/release",
        "arn:aws:glue:us-west-2:913550007193:table/*"
      ]
    }
  ]
}
```
  </div>
</details>
```

(I think this is the right syntax for the docs.)